e2e/e2eApi.ts

```diff
@@ -0,0 +1,74 @@
+import { Page, Browser, launch } from 'puppeteer'
+import config from '../config'
+
+const serverUrl = `http://${config.server_host}:${config.e2e_port}`
+const launchOptions = {
+  dumpio: false,
+  headless: true,
+  slowMo: 10,
+
+  // Workaround for newPage hang in CircleCI: https://github.com/GoogleChrome/puppeteer/issues/1409#issuecomment-453845568
+  args: [process.env.CI && '--single-process'].filter(Boolean),
```

Comment:
We already have `'--single-process'` in two files, can we move this to a shared config?

Reply:
yes - actually, now it is three places (so it was two before), and this is the reason I've followed the pattern :) will address it